qadence_protocols/mitigations/twirl.py

```diff
@@ -55,10 +44,14 @@ def mitigate(
     Returns:
         Mitigated output is returned
     """
+    block = model._circuit.abstract.block
```

Comment:
You might need `circuit.original` here too if you want to work with non-transpiled and non-converted circuits (this is what you did for the observables).

Reply:
both work, but i changed it to original